packages/manager/apps/pci-ai-tools/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@ovh-ux/manager-pci-ai-tools-app",
+  "version": "1.0.0",
+  "private": true,
+  "description": "pci-ai-tools",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/pci-ai-tools"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "type": "module",
+  "scripts": {
+    "build": "tsc --project tsconfig.build.json && vite build",
+    "coverage": "vitest run --coverage",
+    "dev": "tsc && vite",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-pci-ai-tools-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-pci-ai-tools-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-pci-ai-tools-app' --include-dependencies -- npm run dev:watch --if-present",
+    "test": "vitest run",
```

Comment:
You can improve performance by adding `--pool=threads` :)
https://vitest.dev/guide/improving-performance#pool